env.sh

```diff
@@ -1 +1,2 @@
 export SPRING_APPLICATION_JSON=`cat localhost.json`
+export JAVA_HOME=/usr/lib/jvm/java-11-openjdk
```

Comment:
The export JAVA_HOME=/usr/lib/jvm/java-11-openjdk
only works on CSIL so it should NOT go in the env.sh.

It shouldn't be necessary any more as they've fixed CSIL, but if goes anywhere, it goes in your own personal shell startup files.